packages/e2e-tests/cypress/support/common/go-to-page/goToAppealsPage.js

```diff
@@ -1,5 +1,11 @@
-export const goToAppealsPage = (url) =>{
+export const goToAppealsPage = (url, options) =>{
     cy.wrap(`${Cypress.env('APPEALS_BASE_URL')}/${url}`).then((url)=>{
-    cy.visit(url,{failOnStatusCode: false});
+    if(options){
+      options.failOnStatusCode = false;
+    }
+    else {
+      options = {failOnStatusCode: false};
+    }
+      cy.visit(url,options);
```

Comment:
What extra check is this adding?

Reply:
Sorry, don't get you. As the existing code requires failOnStatusCode, that will be added options if it is passed. Otherwise options will have only this property